met-web/src/components/engagement/old-view/EngagementContent.tsx

```diff
@@ -5,16 +5,16 @@ import { ActionContext } from './ActionContext';
 import { Skeleton } from '@mui/material';
 import { getEditorStateFromRaw } from 'components/common/RichTextEditor/utils';
 
-export const EngagementContent = () => {
-    const { richContent, isEngagementLoading } = useContext(ActionContext);
+export const EngagementContent = ({ index = 0 }: { index?: number }) => {
```

Comment:
Thanks for making the old view work with this. Hopefully we can ditch this code soon

Reply:
I hope so too 😆